replicate/secret/secrets_test.go

```diff
@@ -692,6 +765,85 @@ func TestSecretReplicator(t *testing.T) {
 		require.Equal(t, []byte("Hello Bar"), updTarget.Data["bar"])
 	})
 
+	t.Run("replication is pushed to other namespaces and strip labels", func(t *testing.T) {
```

Comment:
To what extend do these tests differ? If they do, please adjust the test description accordingly.

Reply:
They don't.